lib/createAccessorUsageTables.js

```diff
@@ -0,0 +1,168 @@
+                            mappedTableIndex = parseInt(mappedTableIndexArray[0]);
+                            for (var j = 1; j < mappedTableIndexArrayLength; j++) {
+                                // Multiple attributes mapped to different tables, merge them
+                                var indexB = mappedTableIndexArray[j];
+                                mergeUsageTables(tables, accessorTableMapping, mappedTableIndex, indexB);
+                            }
+                        }
+                        if (!defined(mappedTableIndex)) {
+                            // There are no existing linkages to other accessors, make a new table entry
+                            table = {
+                                indexAccessors : {},
+                                attributeAccessors : {},
+                                usage : []
+                            };
+                            tables.push(table);
+                            mappedTableIndex = tables.length - 1;
+                        } else {
+                            // Join an existing mapping
+                            table = tables[mappedTableIndex];
+                        }
+                        table.usage = markUsageForIndexAccessor(gltf, indexAccessor, table.usage);
+                        table.indexAccessors[indexAccessorId] = true;
+                        for (attribute in attributes) {
+                            if (attributes.hasOwnProperty(attribute)) {
+                                attributeAccessorId = attributes[attribute];
+                                table.attributeAccessors[attributeAccessorId] = true;
+                                accessorTableMapping[attributeAccessorId] = mappedTableIndex;
+                            }
+                        }
+                        accessorTableMapping[indexAccessorId] = mappedTableIndex;
+                        mappedTableIndex = undefined;
+                    } else {
+                        // This index accessor has already been done
+                        table = tables[tableIndex];
+                        for (attribute in attributes) {
+                            if (attributes.hasOwnProperty(attribute)) {
+                                attributeAccessorId = attributes[attribute];
+                                // Make sure the attribute accessors haven't been assigned somewhere else
+                                var attributeTableIndex = accessorTableMapping[attributeAccessorId];
+                                if (defined(attributeTableIndex) && attributeTableIndex !== tableIndex) {
+                                    // Merge them into this one if they have been
+                                    mergeUsageTables(tables, accessorTableMapping, tableIndex, attributeTableIndex);
+                                }
+                                table.attributeAccessors[attributeAccessorId] = true;
```

Comment:
Is there any reason you wouldn't be able to use the `attributeAccessors` attribute type as the value instead of just setting it to true? In order to use `createAccessorUsageTables` in PR #100 I would need the attribute types associated with the accessors and it seems like that would be a good spot to sneak them in.
Thanks!